keyboards/handwired/2x5keypad/config.h

```diff
@@ -37,16 +37,16 @@
 /* Locking resynchronize hack */
 #define LOCKING_RESYNC_ENABLE
 
+/* Tap dancing params */
+#define TAPPING_TIME 175
```

Comment:
I think you meant this to be `TAPPING_TERM`; this define doesn't actually exist.